lib/generators/paper_trail/install/templates/create_versions.rb.erb

```diff
@@ -3,7 +3,7 @@
 class CreateVersions < ActiveRecord::Migration<%= migration_version %>
 
   # The largest text column available in all supported RDBMS is
-  # 1024^3 - 1 bytes, roughly one gibibyte.  We specify a size
+  # 1024^3 - 1 bytes, roughly one gigabyte.  We specify a size
```

Comment:
"gibibyte" is the correct term for 1024^3 bytes. (https://en.wikipedia.org/wiki/Gibibyte) "gigabyte" would be 1000^3 bytes.